browser-test/src/admin/admin_predicates_expanded.test.ts

```diff
@@ -2,7 +2,153 @@ import {expect, test} from '../support/civiform_fixtures'
 import {enableFeatureFlag, loginAsAdmin, validateScreenshot} from '../support'
 import {waitForHtmxReady} from '../support/wait'
 import {QuestionType} from '../support/admin_questions'
-import {Locator} from 'playwright'
+import {MultiValueSpec} from '../support/admin_predicates'
+
+/**
+ * Map of question types to that question type's corresponding testing data
+ *    @param questionName: The (backend, admin) name of the question
+ *    @param questionText: The question text displayed to the applicant
+ *    @param firstValue: The default value to fill in an input field, or to be selected from a dropdown
+ *    @param secondValue: The default value to fill in a second input field. Optional, for question types that support BETWEEN operators.
+ *    @param defaultInputType: The default input type for the question type. Optional, for question types that use input tags.
+ *    @param defaultInputMode: The default inputmode for the question type. Optional, for question types that use input tags.
+ */
+const PROGRAM_SAMPLE_QUESTIONS = new Map<
```

Comment:
I made this a global for the test suite, since it's used in a couple of cases already and I plan to use it for the tests for reloading saved predicates.